extensions/wikia/AdEngine/js/config/desktop.js

```diff
@@ -106,6 +108,17 @@ define('ext.wikia.adEngine.config.desktop', [
 			return [adProviderTaboola];
 		}
 
+		// MonetizationService
+		if (context.providers.monetizationService &&
+			adProviderMonetizationService && adProviderMonetizationService.canHandleSlot(slotName)
```

Comment:
Since adProviderMonetizationService is a required module it's always gonna be truthy, so no need to check for it in the condition.